release_notes/ocp_3_10_release_notes.adoc

```diff
@@ -235,73 +319,73 @@ features marked *GA* indicate _General Availability_.
 |TP
 
 |xref:../install_config/master_node_configuration.adoc#master-node-config-audit-config[Central Audit]
-| -
 |TP
 |GA
+|-
 
 |xref:../admin_guide/managing_networking.adoc#enabling-static-ips-for-external-project-traffic[Static IPs for External Project Traffic]
-| -
 |TP
 |GA
+|-
 
 |xref:../dev_guide/templates.adoc#waiting-for-template-readiness[Template Completion Detection]
-| -
 |TP
 |GA
+|-
 
 |xref:../cli_reference/basic_cli_operations.adoc#object-types[`replicaSet`]
 |TP
-|TP
 |GA
+|-
 
 |xref:../install_config/aggregate_logging.adoc#aggregated-fluentd[Mux]
-| -
+|TP
 |TP
 |TP
 
 |Clustered MongoDB Template
-|TP
 |Community
-| -
+|-
+|-
 
 |Clustered MySQL Template
-|TP
 |Community
-| -
+|-
+|-
 
 |xref:../dev_guide/managing_images.adoc#using-is-with-k8s[Image Streams with Kubernetes Resources]
 |TP
-|TP
 |GA
+|-
 
 |xref:ocp-310-device-manager[Device Manager]
-| -
-| -
+|-
 |TP
+|GA
 
 |xref:ocp-310-pv-resize[Persistent Volume Resize]
-| -
-| -
+|-
+|TP
 |TP
 
 |xref:ocp-310-hugepages[Huge Pages]
```

Comment:
@derekwaynecarr Are Huge Pages being promoted to supported in 3.10, or will they remain in Tech Preview?

Reply:
Oh, I see now that they are: https://github.com/openshift/openshift-docs/pull/8706